crates/criticaltrust/src/v2/manifests.rs

```diff
@@ -0,0 +1,177 @@
+// SPDX-FileCopyrightText: The Ferrocene Developers
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+//! Serializable and deserializable representation of criticaltrust v2 manifests.
+
+use serde_semver::SemverReq;
+
+use crate::keys::KeyRole;
+use crate::signatures::Signable;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, PartialEq, Eq, SemverReq)]
+#[version("0.0.1")]
+pub struct MetadataVersion;
+
+impl std::fmt::Debug for MetadataVersion {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_str(&format!("MetadataVersion: {}", MetadataVersion::version())[..])
+    }
+}
+
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "kebab-case")]
+pub struct Metadata {
+    pub release_identifier: String,
+    // TODO: channel should be an enum?
```

Comment:
No, each release creates a new channel

Reply:
There is an enum for this in https://github.com/ferrocene/publish-release/blob/main/src/metadata/build.rs#L46-L51